src/main/java/hudson/plugins/sonar/MsBuildSQRunnerBegin.java

```diff
@@ -67,13 +69,20 @@
    * @since 2.4.3
    */
   private String msBuildScannerInstallationName;
-  private final String sonarInstallationName;
-  private final String projectKey;
-  private final String projectName;
-  private final String projectVersion;
-  private final String additionalArguments;
+  private String sonarInstallationName;
+  private String projectKey;
+  private String projectName;
+  private String projectVersion;
+  private String additionalArguments;
 
   @DataBoundConstructor
+  public MsBuildSQRunnerBegin() {
+  }
+
+  /**
+   * @deprecated since 2.5. Moved to use {@link DataBoundSetter}
+   */
+  @Deprecated
   public MsBuildSQRunnerBegin(String msBuildScannerInstallationName, String sonarInstallationName, String projectKey, String projectName, String projectVersion,
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)